timeboost-builder/src/certifier.rs

```diff
@@ -163,6 +165,19 @@ impl Certifier {
     }
 }
 
+fn translate_addr(c: AddressableCommittee) -> AddressableCommittee {
+    let committee = c.committee().clone();
+    let shifted_entries = c
+        .entries()
+        .map(|(pk, dh, addr)| {
+            let cert_port = addr.port().saturating_add(2000);
```

Comment:
The certifier port should be `CERTIFIER_PORT_OFFSET` away from the public network address port (i.e. the sailfish port).